sample-apps/react/react-video-demo/src/hooks/useEdges.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import { StreamVideoClient } from '@stream-io/video-react-sdk';
+import { FeatureCollection, Geometry } from 'geojson';
+import { createGeoJsonFeatures } from '../utils/useCreateGeoJsonFeatures';
+
+const EDGES_KEY = '@react-video-demo/edges';
+const oneDay = 24 * 3600 * 1000;
+
+type CachedEdges = {
+  edges: FeatureCollection<Geometry>;
+  expires_at: number;
+};
+
+const cacheEdges = (edges: FeatureCollection<Geometry>) => {
+  const payload: CachedEdges = {
+    edges,
+    expires_at: new Date().getTime() + oneDay,
+  };
+  localStorage.setItem(EDGES_KEY, JSON.stringify(payload));
+
+  return payload;
+};
+
+const getCachedEdges = (): CachedEdges | undefined => {
+  const edgeJSON = localStorage.getItem(EDGES_KEY);
+  if (!edgeJSON) return;
+  try {
+    return JSON.parse(edgeJSON);
+  } catch {
+    return undefined;
+  }
+};
+
+export const useEdges = (client?: StreamVideoClient) => {
+  const [edges, setEdges] = useState<CachedEdges | undefined>(getCachedEdges);
+  const [fastestEdge] = useState<{
+    id: string;
+    latency: number;
+  }>();
```

Comment:
Fastest edge is never initialized? What is the idea behind this?

Reply:
This is where I keep the original logic :) I do not know what is the purpose of this variable.